test/onnx/test_models_onnxruntime.py

```diff
@@ -4,16 +4,19 @@
 import os
 import unittest
 from collections import OrderedDict
-from typing import Any, List, Mapping, Tuple, Type
+from typing import List, Mapping, Tuple
 
 import parameterized
 import PIL
 import test_onnx_common
 import torchvision
 from test_models import TestModels
-from test_pytorch_common import TestCase, run_tests, skipIfUnsupportedMinOpsetVersion
-from test_pytorch_common import skipScriptTest_New as skipScriptTest
-from test_pytorch_onnx_onnxruntime import run_model_test
+from test_pytorch_common import (
+    TestCase,
```

Comment:
It's good time we import from common_utils instead:

```python
from torch.testing._internal import common_utils
```

and do 

```
class TestVerification(common_utils.TestCase):
```
TestCase was transitively imported in test_pytorch_common and we should avoid that. related #79543

Reply:
let's leave this to the other PR then. The diff is from unrelated changes on `run_model_test`.